en/core-api/e-wallet.md

```diff
@@ -368,23 +440,51 @@ You needs to prepare a `callback_url` which accept two query parameters.
 ?> **Important!** <br> To update transaction status on your backend/database, DO NOT solely rely on frontend callbacks! For security reason to make sure the status is authentically coming from Midtrans, only update transaction status based on [HTTP Notification](#_3-handling-post-transaction) or 
 [API Get Status](https://api-docs.midtrans.com/#get-transaction-status)
 
-## 3. Handling Post-Transaction
+### 3. Handling Post-Transaction
+
+When the transaction status changes, Midtrans notifies you at the redirect URL and sends HTTP notification to the merchant backend. This ensures that you are updated of the transaction status securely.
+
+HTTP POST request with JSON body will be sent to your *Payment Notification URL* configured on dashboard.
+
+<details>
+<summary><b>Configuring Payment Notification URL</b></summary>
+<article>
+To configure the Payment Notification URL, follow the steps given below.
+
+1. Login to your MAP account.
 
-Other than customer being redirected, when the status of payment is updated/changed (i.e: payment has been successfully received), Midtrans will send **HTTP Notification** (or webhook) to your server's `Notification Url` (specified on Midtrans Dashboard, under menu **Settings > Configuration `Notification URL`**). Follow this link for more details:
+2. On the Home page, go to **SETTINGS > CONFIGURATION**.
+   *Configuration* page is displayed.
+
+3. Enter **Payment Notification URL**.
+
+4. Click **Update**.
+   A confirmation message is displayed.
+
+   ![Core API](./../../asset/image/coreapi/core-api-payment-notification-1.png)
+
+   The *Payment Notification URL* is configured.
+
+   </article>
+   </details>
 
 <div class="my-card">
 
+Follow this link for more details:
+
+
 #### [Handling Webhook HTTP Notification](/en/after-payment/http-notification.md)
 </div>
```

Comment:
This is still poorly rendered, can you fix it? Copy from rashmi's work if necessary on this part